src/serial/private/serialport/serialport_windows.nim

```diff
@@ -72,8 +72,6 @@ proc initDcb(port: SerialPort | AsyncSerialPort, baudRate: int32, parity: Parity
     port.dcb.StopBits = TWOSTOPBITS
   of StopBits.OnePointFive:
     port.dcb.StopBits = ONE5STOPBITS
-  else:
```

Comment:
Is this intentional? The original idea was to make it obvious if an invalid stop bits value was used. I believe the posix version does something similar (off the top of my head).

Reply:
Actually, the posix version doesn't raise an exception either, so this is fine.

Reply:
Yeah, Nim was complaining because the else clause was unreachable due to the fact that all cases were already handled.  It prevented compilation.  I assumed that it was a new restriction in Nim 1.0

Reply:
Yeah, certainly seems like it. I've just removed that exception as it isn't used at all now.